server/src/main/java/io/unitycatalog/server/persist/dao/RegisteredModelInfoDAO.java

```diff
@@ -44,6 +44,9 @@ public class RegisteredModelInfoDAO extends IdentifiableDAO {
   @Column(name = "url", length = 2048)
   private String url;
 
+  @Column(name = "max_version_number")
+  private Long maxVersionNumber;
```

Comment:
`currentVersion` ?

Reply:
current version would indicate what the latest version available is.  Users could delete the latest version but we would not want to reuse that version number.

Reply:
cool .. can we remove the `number` suffix in the variable name  .. it's redundant.

Reply:
So actually....this is a little interesting....in MLflow, we refer to version number as the version (I think they are used interchangeably).  I can remove this if you feel strongly on it, but that is a name that we use versus just being redundant info.

Reply:
Not too strongly :) . Just a suggestion.